pages/testimonials.md

```diff
@@ -1,13 +0,0 @@
----
-layout: page-fullwidth
-permalink: /testimonials/
```

Comment:
We have a few now, we could keep this section

Reply:
Opened #23.